src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -45,6 +45,33 @@ export default class NavigationLibrary {
     }
 
     /**
+     *
+     * @property drawSids
+     * @return
+     */
+    get drawSids() {
```

Comment:
Good callout here.  I put this one in quick to get stuff working with the `CanvasController`.  I'll update the name cause I'm not a fan either.